kujira/rest/controllers/mons.py

```diff
@@ -0,0 +1,40 @@
+from kujira.blueprints import mon_bp
+from kujira.rest.lib.request_methods import send_get_alt
+from kujira.rest.lib.parsing_methods import parse_and_return
+
+
+@mon_bp.route("/<fsid>")
+def all_monitors(fsid):
+    response = send_get_alt('cluster/' + fsid + '/mon')
+    return parse_and_return(mons_parse, response)
+
+
+@mon_bp.route("/<fsid>/<name>")
+def monitor(fsid, name):
+    response = send_get_alt('cluster/' + fsid + '/mon/' + name)
+    return parse_and_return(mons_parse, response)
+
+
+@mon_bp.route("/<fsid>/<name>/status")
+def monitor_status(fsid, name):
+    response = send_get_alt('cluster/' + fsid + '/mon/' + name + 'status')
+    return parse_and_return(mons_parse, response)
+
+
+def mons_parse(json_dict):
+    try:
+        new_dict = json_dict[0]
+    except Exception as e:
+        new_dict = json_dict
+        print e.message
```

Comment:
Do not use "print", use logging.  Use: http://plumberjack.blogspot.com/2010/09/configuring-logging-for-web.html